docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -0,0 +1,573 @@
+
+1. Import indexes and triggers using the `import schema` command with an additional `--post-import-data` flag as follows:
+
+    ```sh
+    # Replace the argument values with those applicable for your migration.
+    yb-voyager import schema --export-dir <EXPORT_DIR> \
+            --target-db-host <TARGET_DB_HOST> \
+            --target-db-user <TARGET_DB_USER> \
+            --target-db-password <TARGET_DB_PASSWORD> \ # Enclose the password in single quotes if it contains special characters.
+            --target-db-name <TARGET_DB_NAME> \
+            --target-db-user <TARGET_DB_USER> \
+            --target-db-schema <TARGET_DB_SCHEMA> \
+            --post-import-data
+    ```
+
+    Refer to [import schema](../../reference/yb-voyager-cli/#import-schema) for details about the arguments.
+
+1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and target database to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
+
+    {{< warning title = "Caveat associated with rows reported by import data status" >}}
+
+Suppose you have a scenario where,
+
+- [import data](#import-data) or [import data file](#import-data-file) command fails.
+- To resolve this issue, you delete some of the rows from the split files.
+- After retrying, the import data command completes successfully.
+
+In this scenario, [import data status](#import-data-status) command reports incorrect imported row count; because it doesn't take into account the deleted rows.
+
+For more details, refer to the GitHub issue [#360](https://github.com/yugabyte/yb-voyager/issues/360).
+
+    {{< /warning >}}
+
+1. Stop [archive changes](#archive-changes).
+
+### Fall-forward switchover (Optional)
+
+During this phase, switch over your application from pointing it to the target YugabyteDB to pointing it to the fall-forward database. As this step is optional, perform it _only_ if the target YugabyteDB is not working as expected.
+
+Keep monitoring the metrics displayed for `fall-forward synchronize` and `fall-forwad setup` processes. After you notice that the import of events to the fall-forward database is catching up to the exported events from the target database, you are ready to cutover. You can use the "Remaining events" metric displayed in the import data process to help you determine the cutover.
+
+Perform the following steps as part of the cutover process:
+
+1. Quiesce your source database, that is stop application writes.
+1. Initiate switchover as follows:
+
+    1. After the exported events rate ("ingestion rate" in the metrics table) drops to 0, it is safe to switchover using the following command:
+
```

Comment:
cut over to a database
switch over your database
perform a cutover
perform a switchover
also, why are we using both? can clean this up later though

Reply:
Switchover is needed only if the target yugabyteDB database is not working as expected. So its an option for such a case.

Reply:
This was just a note on using the terms, that is, 'cut over' when a verb, and 'cutover' when a noun
So we are distinguishing between cutover from Oracle to YBDB, and switchover from YBDB to fall forward YBDB?

Reply:
Got it. Thanks!